db/migrate/20150916162754_add_log_path_to_services_table.rb

```diff
@@ -0,0 +1,5 @@
+class AddLogPathToServicesTable < ActiveRecord::Migration
+  def change
+    add_column :services, :log_path, :text
+  end
```

Comment:
@xbili The log path is stored in log_paths.yml since it's common for all logs of a particular service, eg nginx. Also, having all logs in /var/log makes sure that we need to give the lager user, read permissions only to that directory. 

If they have it in a different place, they can just symlink it there. What do you think?